packages/ui/tsconfig.json

```diff
@@ -5,25 +5,21 @@
     }
   },
   "compilerOptions": {
-    "target": "es5",
-    "module": "esnext",
-    "strict": false,
-    "jsx": "preserve",
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": ".",
+    "declaration": true,
+    "esModuleInterop": true,
     "importHelpers": true,
+    "lib": ["esnext", "dom", "dom.iterable", "scripthost"],
+    "module": "esnext",
     "moduleResolution": "node",
+    "outDir": "dist/esm",
+    "paths": { "@/*": ["src/*"] },
     "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "allowJs": true,
     "sourceMap": true,
-    "declaration": true,
-    "outDir": "dist",
-    "baseUrl": ".",
-    "types": ["webpack-env", "jest"],
-    "paths": {
-      "@/*": ["src/*"]
-    },
-    "lib": ["esnext", "dom", "dom.iterable", "scripthost"]
+    "strict": false,
+    "target": "es5",
```

Comment:
Thinking if we should update the target to align other consumer packages. Those customer facing packages target a higher ES version which makes es5 looks dumb here.

Reply:
Agree but believe that update has the potential to be a breaking change